### PR TITLE
Add Obro

### PR DESCRIPTION
**Pull request by holtenc**
Add obro keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/obro/rules.mk [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes      # Audio control and System control
+CONSOLE_ENABLE = yes       # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes          # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/handwired/obro/rules.mk:16]:
```suggestion
NKRO_ENABLE = yes          # Enable N-key Rollover
```

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/obro/config.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#define MATRIX_COLS 14
+
+#define MATRIX_COL_PINS { C7, C6, B6, B5, B4, F7, D4, B7, B3, D5, D3, D2, D1, D0 }
+#define MATRIX_ROW_PINS { E6, F0, F6, D7, D6 }
+
+#define DIODE_DIRECTION COL2ROW
+
+#define RGB_DI_PIN F1
+#define RGBLED_NUM 64
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/handwired/obro/config.h:40]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/obro/rules.mk [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic
+MOUSEKEY_ENABLE = no       # Mouse keys
+EXTRAKEY_ENABLE = yes      # Audio control and System control
+CONSOLE_ENABLE = yes       # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/handwired/obro/rules.mk:15]:
```suggestion
```

File: keyboards/handwired/obro/readme.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/obro:default
+    make handwired/obro:via
+
+Flashing example for this keyboard:
+
+    make handwired/obro:default:flash
+    make handwired/obro:via:flash
```
Comment (fauxpark) [inline keyboards/handwired/obro/readme.md:15]:
```suggestion

Flashing example for this keyboard:

    make handwired/obro:default:flash
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**